domain/src/test/java/org/oppia/android/domain/platformparameter/syncup/PlatformParameterSyncUpWorkerTest.kt

```diff
@@ -327,17 +318,24 @@ class PlatformParameterSyncUpWorkerTest {
     assertThat(workInfo.get().state).isEqualTo(WorkInfo.State.FAILED)
 
     val exceptionMessage = fakeExceptionLogger.getMostRecentException().message
-    assertThat(exceptionMessage).isEqualTo(
-      PlatformParameterSyncUpWorker.EMPTY_RESPONSE_EXCEPTION_MSG
-    )
+    assertThat(exceptionMessage)
+      .isEqualTo(PlatformParameterSyncUpWorker.EMPTY_RESPONSE_EXCEPTION_MSG)
+
     // Retrieve the previously cached Platform Parameters from Cache Store.
     platformParameterController.getParameterDatabase().toLiveData().observeForever(mockUnitObserver)
     testCoroutineDispatchers.runCurrent()
 
     // Values retrieved from Cache store will be sent to Platform Parameter Singleton by the
     // Controller in the form of a Map, therefore verify the retrieved values from that Map.
     val platformParameterMap = platformParameterSingleton.getPlatformParameterMap()
-    assertThat(platformParameterMap).isNotEmpty()
+
+    // Previous String Platform Parameter is still same in the Database.
+    assertThat(platformParameterMap)
+      .containsEntry(TEST_STRING_PARAM_NAME, expectedTestStringParameter)
+
+    // Previous Integer Platform Parameter is still same in the Database.
+    assertThat(platformParameterMap)
+      .containsEntry(TEST_INTEGER_PARAM_NAME, defaultTestIntegerParameter)
```

Comment:
Suggest using containsExactly() to fully verify the map & ensure nothing else is included.

Reply:
As the map is supposed to have more than two platforms here so containsExactly will not serve the purpose.